content/auth/revocation.textile

```diff
@@ -82,76 +82,81 @@ The following target specifier can also be enabled for you by Ably, should your
 
 * channel
 
-h3(#client-id). clientId 
+h3(#client-id). clientId
 
-This target specifier will match tokens that have the specified @clientId@. 
+This target specifier will match tokens that have the specified @clientId@.
 
 For example, @targets: ["clientId:client1"]@ matches tokens containing the @clientId@ of @client1@.
 
 A sample request to revoke an Ably token based on @clientId@, using the REST interface of an SDK:
 
 ```[rest_javascript]
-const Ably = require('ably');
+const Ably = require('ably/promises');
 
-const ablyRest = new Ably.Rest({ key: '{{API_KEY}}' });
+const ablyRest = new Ably.Rest.Promise({ key: '{{API_KEY}}' });
 const requestBody = { targets: ['clientId:client1@example.com'] };
 
-ablyRest.request('post', '/keys/{{API_KEY_NAME}}/revokeTokens', null, requestBody, null,
-
-  function (err, response) {
-    console.log(response);
-    if (!response.success) {
-      console.log('An error occurred; err = ' + response.errorMessage);
-    } else {
-      console.log('Success! status code was ' + response.statusCode);
-    }
-  }
+const revocationResponse = await ablyRest.request(
+  'post',
+  '/keys/{{API_KEY_NAME}}/revokeTokens',
+  null,
+  requestBody,
+  null
```

Comment:
this argument is optional so can be removed (same goes for the below snippet)

Reply:
Removed the unnecessary argument. https://github.com/ably/docs/pull/2119/commits/e90046ac431bd699719987715d1edab1ac28e59d